index.js

```diff
@@ -1,5 +1,100 @@
 'use strict';
 
+const Funnel = require('broccoli-funnel');
+const path = require('path');
+
+const componentDependencies = {
+  'uni-auth-modal': ['uni-modal'],
+  'uni-autocomplete': ['uni-autocomplete-no-results'],
+  'uni-checkbox': ['uni-input-container'],
+  'uni-country': ['uni-autocomplete'],
+  'uni-datepicker-input': ['uni-datepicker'],
+  'uni-file-upload': ['uni-button'],
+  'uni-form': ['uni-button'],
+  // @TODO: Check if 'uni-info-card' is still used in any platform
+  // If not, kill it with fire.
+  'uni-info-card': ['uni-modal'],
+  'uni-mobile-number': ['uni-select', 'uni-tooltip'],
+  'uni-modal': ['uni-title'],
+  'uni-multi-selector': ['uni-selector-button'],
+  'uni-radio-button': ['uni-input-container'],
+  'uni-textarea': ['uni-tooltip'],
+  'uni-title': ['uni-subtitle']
+}
+
 module.exports = {
-  name: 'ember-cli-uniq'
+  name: require('./package').name,
+
+  included() {
+    this._super.included.apply(this, arguments);
```

Comment:
no easy way to do this?
and out of curiosity what would be the super of this?

Reply:
I don't think so, at least all addons I've checked do it that way, e.g. https://github.com/DockYard/ember-composable-helpers/blob/master/index.js#L12

Reply:
I've checked the API documentation and it has the same syntax: https://ember-cli.com/api/classes/Addon.html (search for `public included(parent)`)

Reply:
Thanks, @AndreJoaquim 🙏 !!!